oneanddone/tasks/views.py

```diff
@@ -350,19 +351,125 @@ def post(self, *args, **kwargs):
-    allow_expired_tasks = True
+class TaskAttemptDetailView(LoginRequiredMixin,
+                            MyStaffUserRequiredMixin,
+                            generic.View):
+
+    def get(self, request, *args, **kwargs):
+        view = TaskAttemptDisplayView.as_view()
+        return view(request, *args, **kwargs)
+
+    def post(self, request, *args, **kwargs):
+        view = SubmitAdminVerificationResponseView.as_view()
+        return view(request, *args, **kwargs)
+
+
+class TaskAttemptDisplayView(LoginRequiredMixin,
+                             MyStaffUserRequiredMixin,
+                             generic.DetailView):
+    model = TaskAttempt
+    template_name = 'tasks/attempt_detail.html'
 
     def get_context_data(self, *args, **kwargs):
-        ctx = super(TaskDetailView, self).get_context_data(*args, **kwargs)
+        ctx = super(TaskAttemptDisplayView, self).get_context_data(*args, **kwargs)
+        ctx['form'] = SubmitVerifiedTaskForm()
+        return ctx
+
+
+class SubmitAdminVerificationResponseView(LoginRequiredMixin,
+                                          MyStaffUserRequiredMixin,
+                                          generic.detail.SingleObjectMixin,
+                                          generic.FormView):
+
+    form_class = SubmitVerifiedTaskForm
+    model = TaskAttempt
+    template_name = 'tasks/attempt_detail.html'
+
+    def post(self, request, *args, **kwargs):
+        self.object = self.get_object()
+        return super(SubmitAdminVerificationResponseView, self).post(request, *args, **kwargs)
+
+    def form_valid(self, form):
+
+        attempt = self.get_object()
+
+        communication = form.save(commit=False)
+        communication.attempt = attempt
+        communication.type = TaskAttemptCommunication.ADMIN
+        communication.creator = self.request.user
+        communication.save()
+
+        # Send email to attempt user
+        if attempt.user.profile.consent_to_email:
+            task_name = attempt.task.name
+            subject = 'One and Done task verification update for %s' % task_name
+            link_prefix = 'http'
+            if self.request.is_secure():
+                link_prefix += 's'
+            task_link = link_prefix + '://%s%s' % (
+                self.request.get_host(),
+                attempt.task.get_absolute_url())
+            template = get_template('tasks/emails/verification_email_admin.txt')
+
+            message = template.render({
+                'task_name': task_name,
+                'task_link': task_link,
+                'content': communication.content})
+
+            # Manually replace quotes and double-quotes as these get
+            # escaped by the template and this makes the message look bad.
+            filtered_message = message.replace('&#34;', '"').replace('&#39;', "'")
+
+            send_mail(
+                subject,
+                filtered_message,
+                'oneanddone@mozilla.com',
```

Comment:
As mentioned above, I need to figure out how site-specific settings are going to work with the new platform for the site. Stackato uses environment variables, but I think we'll need to move to something else. I will keep this in mind when I do that work.